packages/web-domains/src/common/apis/base.api.ts

```diff
@@ -0,0 +1,41 @@
+import axios, { AxiosRequestConfig, AxiosInstance } from 'axios';
+const baseURL = `/api`;
+
+export class Http {
+  private instance: AxiosInstance;
+  constructor(path: string) {
+    path = path.startsWith('/') ? path.slice(1) : path;
+
+    this.instance = axios.create({
+      baseURL: `${baseURL}/${path}`,
+    });
+  }
+
+  protected async GET<Response = unknown>(url: string, config?: AxiosRequestConfig) {
+    const response = await this.instance.get<Response>(url, config);
+    return response.data;
+  }
```

Comment:
```suggestion
  protected async GET<T>(args: Parameters<typeof this.instance.get>) {
    const response = await this.instance.get<T>(...args);
    return response.data;
  }
```

요렇게는 어떠신가요? 최대한 http 메서드 인터페이스 따라갔으면 하는 생각입니당 🏃

Reply:
아주 좋습니다. 이렇게 할 생각을 못했어요

Reply:
https://github.com/depromeet/15th-team3-FE/pull/57/commits/815f7da75c394054b8389fb1d9b40f4d62b71df9#diff-302e37b0440d1384c7fb9fb444a7f4cab9647ce762f7dfe5267a50780b7b65d2R9

오 변경하신 거 확인했는데 `<Response = unknown>`으로 하신 이유 있나용? 만약 제네릭으로 타입을 넘기지 않으면 unknown으로 추론될텐데, 저는 `= unknown` 제거해서 엄격하게 타입 지정하도록 강제하는 것도 좋을 거 같은데 어떠신가요?

Reply:
@Doeunnkimm 
이거 unknown 지정 안해줘도 = unknown 한거랑 같을걸요?
실제 결과 비교한것도 같고요.  unknown 지정해준건 마우스 올렸을 때, 기본값이 unkown이라는걸 제대로 표시해주려고 였긴합니다.
혹시 제네릭에 타입 안넘겨주면, 빨간줄 뜨게 해줄수도 있나요? 제네릭 타입 그나마 엄격하게 강제해주는건 extends 사용해서 제약 걸어주는걸 알고 있긴한데..
<img width="324" alt="스크린샷 2024-07-14 오후 4 52 23" src="https://github.com/user-attachments/assets/42d8b711-3526-405a-bb19-4174b9fe75e6">
<img width="324" alt="스크린샷 2024-07-14 오후 4 52 28" src="https://github.com/user-attachments/assets/25fb2f0c-dceb-4353-ad40-30519e5e29f3">

Reply:
아하 그렇군영 ! 고우~